index.js

```diff
@@ -0,0 +1,65 @@
+import Book from './modules/book.js';
+
+import { DateTime } from './modules/luxon.js';
+
+document.addEventListener('DOMContentLoaded', Book.displayBooks());
+
+const form = document.querySelector('#bookForm');
+
+form.addEventListener('submit', () => {
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+
+  const book = new Book(title, author);
+
+  Book.addBookToList(book);
+  Book.addBook(book);
+  form.reset();
+});
+
+document.querySelector('#bookOfList').addEventListener('click', (e) => {
+  Book.deleteBook(e.target);
+  Book.removeBook(e.target.previousElementSibling.textContent);
+});
+
+// Get current date and time and display it
+const now = new Date();
+const date = DateTime.now().toLocaleString(DateTime.DATE_FULL);
+const time = now.toLocaleTimeString();
+document.querySelector('#date').textContent = `${date} ${time}`;
+
+//  navigation
+
+// const bookListSection = document.getElementById('booklist');
+// const addNewBookSection = document.getElementById('addnewbook');
+// const contactSection = document.getElementById('contact');
```

Comment:
- [ ] Great work so far, I would recommend removing these commented code as mentioned in the best practices it is not recommended to commit commented pieces of code to the codebase :+1:

Reply:
Resolved, thank you